### PR TITLE
Create govready-0.9.yaml

### PR DESCRIPTION
**Pull request by mgifford**
So should the PR be just the .yaml file?  Not sure if I should create the .html & .md first.

### CODE REVIEW

**Comment by mgifford** [comment]
If you could look at this, that would be great. Thanks @dmundra

File: opat/govready-0.9.yaml [hunk 1]
```diff
@@ -0,0 +1,698 @@
...
+legal_disclaimer: "The information herein is provided in good faith based on the analysis of the web application at the time of the review and does not represent a legally-binding claim. Please contact us to report any accessibility errors or conformance claim errors for re-evaluation and correction, if necessary."
+chapters:
+  success_criteria_level_a:
+    notes: "GovReady-Q is a web application and an authoring tool. As such the Software & Authoring Tool Notes are all included within the Web for the 'Conformance Level' and 'Remarks and Explanations'. There are some elements here that clearly fail WCAG 2.0 A."
+    criteria:
+      - num: "1.1.1"
+        components:
+          - name: "web"
+            adherence:
+              level: "Does Not Support"
```
Comment (dmundra) [inline opat/govready-0.9.yaml:34]: @mgifford this should be

```
level: does-not-support
```

Comment (mgifford) [reply opat/govready-0.9.yaml]: That's annoying. How did I miss that...

Comment (mgifford) [reply opat/govready-0.9.yaml]: Ok, now it is valid.

File: opat/govready-0.9.yaml [hunk 2]
```diff
@@ -0,0 +1,698 @@
...
+  email: gregelin@govready.com
+  phone: 202-505-1050
+  website: https://govready.com/
+
+notes: "All user interfaces for the GovReady software are done through the web interface, so the focus is on the WCAG 2.0 Report.  Chapter 5  has been deleted as it is redundant.  GovReady is built on the Django framework and some of the accessibility issues have been inherited from this framework. GovReady is an open source tool with an open issue queue.
+Accessibility issues identified here will be documented as accessibility issues within existing issue queues. Any workarounds will be available within the issue queue until such time as the problem is resolved. The Authoring Tools components, 504, have not been reviewed in this assessment.
+Content is generally saved in a machine readable format that allows for it to be processed in many ways. All GovReady documentation is available through the web interface, so the focus is on the WCAG 2.0 Report.  Chapter 6 has been deleted as it is redundant. The documentation is being served through Read the Docs and with their default theme. Some accessibility issues have been inherited from this framework."
```
Comment (dmundra) [inline opat/govready-0.9.yaml:23]: @mgifford `pre-commit` is complaining about this multi-line string indentation. Do you want to try to indent it or use the multi-line string YAML syntax?

Comment (mgifford) [reply opat/govready-0.9.yaml]: Hopefully that caught it.

**Review by dmundra — APPROVED** [review]
Looking good!

**Pull request merged**